From af46a6d4f31243a842661383dc02548a1b1a3387 Mon Sep 17 00:00:00 2001
From: Jeffrey Cody <jcody@redhat.com>
Date: Mon, 13 Jan 2014 20:53:20 +0100
Subject: [PATCH 13/16] docs: updated qemu-img man page and qemu-doc to reflect VHDX support.

RH-Author: Jeffrey Cody <jcody@redhat.com>
Message-id: <8e219e849b375a123cef62478edc843bb4439841.1389633864.git.jcody@redhat.com>
Patchwork-id: 56644
O-Subject: [RHEL7 qemu-kvm PATCH] docs: updated qemu-img man page and qemu-doc to reflect VHDX support.
Bugzilla: 1017650
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>

The man page for qemu-img, and the qemu-doc, did not mention VHDX
as a supported format.  This adds in reference to VHDX in those
documents.

[Stefan Weil <sw@weilnetz.de> suggested s/Block Size/Block size/ for
consistency.  I have made this change.
--Stefan]

Signed-off-by: Jeff Cody <jcody@redhat.com>
Reviewed-by: Stefan Weil <sw@weilnetz.de>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 8282db1b2e7394574cb55fcc608c5cb0df159d8f)

BZ 1017650
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=6861830

Signed-off-by: Jeff Cody <jcody@redhat.com>
---
 qemu-doc.texi | 15 +++++++++++++++
 qemu-img.texi |  4 ++--
 2 files changed, 17 insertions(+), 2 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qemu-doc.texi |   15 +++++++++++++++
 qemu-img.texi |    4 ++--
 2 files changed, 17 insertions(+), 2 deletions(-)

diff --git a/qemu-doc.texi b/qemu-doc.texi
index 5fc0eae..319c2f2 100644
--- a/qemu-doc.texi
+++ b/qemu-doc.texi
@@ -654,6 +654,21 @@ Supported options:
 Specifies which VHD subformat to use. Valid options are
 @code{dynamic} (default) and @code{fixed}.
 @end table
+
+@item VHDX
+Hyper-V compatible image format (VHDX).
+Supported options:
+@table @code
+@item subformat
+Specifies which VHDX subformat to use. Valid options are
+@code{dynamic} (default) and @code{fixed}.
+@item block_state_zero
+Force use of payload blocks of type 'ZERO'.
+@item block_size
+Block size; min 1 MB, max 256 MB.  0 means auto-calculate based on image size.
+@item log_size
+Log size; min 1 MB.
+@end table
 @end table
 
 @subsubsection Read-only formats
diff --git a/qemu-img.texi b/qemu-img.texi
index da36975..a71f132 100644
--- a/qemu-img.texi
+++ b/qemu-img.texi
@@ -427,8 +427,8 @@ This option can only be enabled if @code{compat=1.1} is specified.
 
 @item Other
 QEMU also supports various other image file formats for compatibility with
-older QEMU versions or other hypervisors, including VMDK, VDI, VHD (vpc), qcow1
-and QED. For a full list of supported formats see @code{qemu-img --help}.
+older QEMU versions or other hypervisors, including VMDK, VDI, VHD (vpc), VHDX,
+qcow1 and QED. For a full list of supported formats see @code{qemu-img --help}.
 For a more detailed description of these formats, see the QEMU Emulation User
 Documentation.
 
-- 
1.7.1

